.github/actions/smoke-test_v2/action.yml

```diff
@@ -0,0 +1,53 @@
+name: smoke-test-v2
+description: runs smoke tests
+
+inputs:
+  environment:
+    description: Environment to run tests in
+    required: true
+  pr-number:
+    description: PR number if testing a review app
+    required: false
+  slack-webhook:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Set up Ruby 3.1.0
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.1.0
+
+    - name: install bundler and gems
+      shell: bash
+      run: |
+        gem install bundler
+        echo 'gem "httparty"' >> Gemfile
+        echo 'gem "rspec"' >> Gemfile
+        bundle
+
+    - name: Set Environment variables
+      shell: bash
+      run: |
+        if [ ! -z "${{ inputs.pr-number }}" ]; then
+          PR_NUMBER=${{ inputs.pr-number }}
+          echo "base_url: https://publish-review-$PR_NUMBER.test.teacherservices.cloud" >> config/settings/review_aks.yml
+          echo "publish_api_url: https://publish-review-$PR_NUMBER.test.teacherservices.cloud" >> config/settings/review_aks.yml
+        fi;
+
+    - name: Smoke Tests ${{ inputs.environment }}
+      shell: bash
+      run: RAILS_ENV=${{ inputs.environment }} ./bin/bundle exec rspec spec/smoke --format documentation
+
+    # - name: 'Notify #twd_findpub_tech on failure'
+    #   if: ${{ failure() && inputs.environment != 'review' }}
+    #   uses: rtCamp/action-slack-notify@master
+    #   env:
+    #     SLACK_CHANNEL: twd_findpub_tech
+    #     SLACK_COLOR: '#ef5343'
+    #     SLACK_ICON_EMOJI: ':github-logo:'
+    #     SLACK_USERNAME: Publish Teacher Training
+    #     SLACK_TITLE: Smoke tests failure
+    #     SLACK_MESSAGE: ':alert: <!channel> Smoke tests failure on ${{ inputs.environment }} :sadparrot:'
+    #     SLACK_WEBHOOK: ${{ inputs.slack-webhook }}
```

Comment:
Leftovers?

Reply:
Yea, I'll comment out when all out testing is done